.github/workflows/main.yml

```diff
@@ -7,6 +7,7 @@ on:
 jobs:
   test:
     strategy:
+      fail-fast: false
```

Comment:
Please can we leave this on. It seems wasteful to run everything if something has failed.

Reply:
Wasteful of compute time or developer time? `fail-fast: false` saves developers time because they see all failing jobs in a single run and can understand the root cause faster and be more efficient in fixing.  If this helps them make fewer commits then it also reduces compute time.

`fail-fast: true` is a good default for GitHub because it reduces compute time however it forces the developer to go thru multiple commits when debugging multiple problems.,